README.md

```
@@ -1,88 +1,108 @@
+<details>
+<summary><strong>🤖 Claude Desktop</strong></summary>
 
-1. Fill the following [form](https://docs.google.com/forms/d/e/1FAIpQLSfScbqRpgx3ZzkCmfVjKs8YogWDshOZW9p-LVXrWzIXjcHKrQ/viewform?usp=dialog) to request an Axiomatic_AI API key.
-2. Once received, add the API key to your MCP client configuration as described in the Configuration section below.
+1. Open Claude Desktop settings → Developer → Edit MCP config
+2. Add this configuration:
 
-## Configuration
+```json
+{
+  "mcpServers": {
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)